goslinks/google_oauth2/utils.py

```diff
@@ -1,6 +1,6 @@
 import google.oauth2.credentials
 import googleapiclient.discovery
-from authlib.client import OAuth2Session
+from authlib.integrations.requests_client import OAuth2Session
```

Comment:
why not use `authlib.integrations.flask_client` directly? For Google login, we have a demo here: https://github.com/authlib/demo-oauth-client/tree/master/flask-google-login

Reply:
Thanks for the tip, that will be way simpler!

Back when I first added that integration, I followed this blog post:
https://www.mattbutton.com/2019/01/05/google-authentication-with-python-and-flask/

You may want to comment there to share this demo with others that stumble on that blog post also.

Reply:
@RevolutionTech thanks for letting me know.